sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -103,8 +106,9 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
           TargetFrameworkIdentifier="$(TargetFrameworkIdentifier)"
           TargetFrameworkVersion="$(TargetFrameworkVersion)"
           OutputPath="$(TargetDir)"/>
-
+        <!-- Do not copy _FunctionsMetadataLoaderExtensionFile when in placeholder mode, as we don't want the FunctionMetadataLoader entry in extensions.json--> 
         <Copy
+          Condition="!$(FunctionsEnablePlaceholder)"
```

Comment:
Does this change also need to be made lower in the file (see [here](https://github.com/Azure/azure-functions-dotnet-worker/blob/c1e4f88e14098c742080f2f29add3db633ef583e/sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets#L294) around line 294-ish).

Reply:
No. This 2 places are enough for NetFx & .net core cases. Tested this version on both project types and working as expected.